src/js/index.js

```diff
@@ -3,16 +3,4 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-/* eslint-disable */
-(function() {
-  /**
-   * Kick off all components
-   */
-  Alert.init();
-  Drawer.init();
-  Dropdown.init();
-  Modal.init();
-  Tabs.init();
-  FileInput.init();
-})();
-/* eslint-enable */
+export * from './components/alert.js';
```

Comment:
Also, just and FYI, Rollup automatically resolves the file paths/type, so you can omit the `.js` extension here, just to clean it up a bit. 👍